tensorflow/python/debug/cli/analyzer_cli_test.py

```diff
@@ -46,6 +46,19 @@
 from tensorflow.python.platform import test
 from tensorflow.python.util import tf_inspect
 
+# There are two types MKL supported operators. One type operators whose kernels
```

Comment:
Thank you very much! I really like your explanation, but I think that is too detailed (and can confuse people who are not familiar with TF-MKL). How about putting the details in `tensorflow/core/graph/mkl_layout_pass.cc` or some other MKL file instead? (Doesn't have to be in this PR. Up to you.) 

For here, I think this is enough.
```python
# Helper function to accommodate MKL-enabled TensorFlow:
# MatMul op is supported by MKL and its name is prefixed with "_Mkl" during the
# MKL graph rewrite pass.
```

Reply:
changed.